group.py

```diff
@@ -2,4 +2,60 @@
 
 # Your code to go here...
 
-my_group =
+my_group = {"jill": {
+                    "age": 26,
+                    "occupation": "biologist",
+                    "friend": "zalika",
+                    "partner": "john",
+                    "cousin": "",
+                    "landlord": "",
+                    "tenant": ""
+                                }, 
+           "zalika": {
+                    "age": 28,
+                    "occupation": "artist",
+                    "friend": "jill",
+                    "partner": "",
+                    "cousin": "",
+                    "landlord": "nash",
+                    "tenant": ""
+                     },
+           "john": {
+                    "age": 27,
+                    "occupation": "writer",
+                    "friend": "",
+                    "partner": "jill",
+                    "cousin": "nash",
+                    "landlord": "",
+                    "tenant": ""
+                     },
+           "nash": {
+                    "age": 34,
+                    "occupation": "chef",
+                    "friend": "",
+                    "partner": "",
+                    "cousin": "john",
+                    "landlord": "",
+                    "tenant": "zalika"}
+           
+                    }
+
+
+ages = [my_group[person]["age"] for person in my_group]
+
+# Assuming relations means has partner or cousin
+partners = ["partner" for person in my_group if my_group[person]["partner"]]
```

Comment:
could you simplify this to 
```suggestion
partners = ["partner" for person in my_group if person["partner"]]
```